### ISSUE TITLE
FluffyChat doesn't expose language option in systen settings

### ISSUE THREAD

**Issue by krille-chan** [krille-chan/fluffychat#334]
#### Description

On iOS, the app language can generally be specified as other than the primary system language in Settings, however FluffyChat doesn't allow this.

Previously reported in https://gitlab.com/famedly/fluffychat/-/issues/746#note_798086929.

#### To Reproduce

1. Open the Settings app
2. Scroll to FluffyChat
3. See that there is no language option.

If you check other apps, a lot have a language option there, for example 1.1.1.1, Adobe Acrobat, AdGuard, Authy, Bitwarden, Brave, and conmsidering I am still listing letter A of apps I consider popular, it's very common and I consider it an issue with FluffyChat.

#### Additional information:

- Device: iPhone SE2020
- OS and OS version: iOS
- Installed version of FluffyChat: 
- (Android only) Which store are you using: Testflight

**Comment by Mikaela** [comment]
I retitled this as I think I was the original opened at GitLab and the issue also affects Android.

Going to Settings -> App Info -> FluffyChat should expose a language setting, but it's not there.

If I look at other apps such as YouTube Music or Headspace, there is a language menu.

**Comment by Mikaela** [comment]
Implementation in Android app Voice Notify in case it's helpful:

- https://github.com/pilot51/voicenotify/commit/65af1cd7559e9326a2fc7fd5bb28d598600873ad

**Comment by Mikaela** [comment]
This seems to be the Android documentation:

- https://developer.android.com/guide/topics/resources/app-languages

**Comment by Mikaela** [comment]
- https://developer.apple.com/news/?id=u2cfuj88

looks like iOS equivalent

**Comment by Mikaela** [comment]
Blocked by https://github.com/flutter/flutter/issues/109842?

**Comment by Mikaela** [comment]
Upstream

**Comment by Mikaela** [comment]
I just marked https://github.com/krille-chan/fluffychat/issues/2147 as a duplicate of this issue four days ago.
